main/lsp/LSPIndexer.cc

```diff
@@ -176,8 +175,6 @@ void LSPIndexer::initialize(LSPFileUpdates &updates, WorkerPool &workers) {
     auto savedErrorQueue = initialGS->errorQueue;
     initialGS->errorQueue = make_shared<core::ErrorQueue>(savedErrorQueue->logger, savedErrorQueue->tracer);
     initialGS->errorQueue->ignoreFlushes = true;
-    // Enforce that this is only run once.
```

Comment:
(We already have an `initialized` boolean that we check in this method, so this ENFORCE was redundant anyway.)